holoviews/plotting/bokeh/element.py

```diff
@@ -104,6 +103,12 @@ class ElementPlot(BokehPlot, GenericElementPlot):
           * timeout   - Timeout (in ms) for checking whether interactive
                         tool events are still occurring.""")
 
+    sizing_mode = param.ObjectSelector(default='fixed',
+                                       objects=[
+            'fixed', 'scale_width', 'scale_height',
+            'scale_both', 'stretch_both'
+        ], doc="""Defines how the plot scales when resized.""")
+
```

Comment:
A WIP tag says that the *pull request* is in work in progress which means there is a public conversation going on about a feature that is in progress. That isn't the same as 'please do not comment' and it sounds like that is how you want to define it.

Github defines a [PR](https://help.github.com/articles/about-pull-requests/) as:

> Pull requests let you tell others about changes you've pushed to a repository on GitHub.

It is clearly there for the purposes of communication, even if you are communicating to yourself (e.g to see an overview of your changes). I'm wondering if there is a tag that better conveys what you intended - not only 'work in progress' but 'don't supply feedback yet'.